Laboratory1/app/src/main/java/com/example/laboratory1/ThirdActivity.kt

```diff
@@ -0,0 +1,87 @@
+package com.example.laboratory1
+
+import android.content.Intent
+import android.os.Build
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.*
+import androidx.annotation.RequiresApi
+import androidx.appcompat.app.AppCompatActivity
+
+
+class ThirdActivity : AppCompatActivity() {
+    @RequiresApi(Build.VERSION_CODES.LOLLIPOP)
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_third)
+
+        val goToListBtn = findViewById<Button>(R.id.goToListBtn)
+        val userName = findViewById<TextView>(R.id.userName)
+        val goToMainBtn = findViewById<Button>(R.id.goToMainBtn)
+        val setAvatar = findViewById<ImageView>(R.id.setAvatar)
+        val yourTxtArea = findViewById<EditText>(R.id.yourTxtArea)
+        val goToRandomBtn = findViewById<Button>(R.id.goToRandomBtn)
+        val button = findViewById<Button>(R.id.button)
+
+//        yourTxtArea.addTextChangedListener(object: TextWatcher{
+//            override fun afterTextChanged(s: Editable?) {
+//                TODO("Not yet implemented")
+//            }
+//
+//            override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {
+//                TODO("Not yet implemented")
+//            }
+//
+//            override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {
+//                val thisInput = yourTxtArea.text.toString().trim()
+//
+//                if (thisInput.isNotEmpty()) {
+//                goToListBtn.isEnabled
+//            }
+//        }})
```

Comment:
Why do you need that dead code? Remove it and you will find it in version control if you will need it later

Reply:
I refactored the code using viewBinding and removed all unnecessary, commented code. Thanks for the feedback!